userguide/system.rst

```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
-(for TCG devices). When managing SED devices from the command line, it is
-important to use :command:`sedutil-cli` rather than camcontrol
-to access the full capabilities of the device. %brand% provides the
-:command:`sedhelper` wrapper script to ease SED device administration from
-the command line.
-
-By default, SED devices are not locked until the administrator takes
-ownership of them. This is done by explicitly configuring a global or
-per-device password in the %brand% |web-ui| and adding the password to
-the SED devices.
-
-Once configured, the system automatically unlocks all SEDs during the boot
-process, without requiring manual intervention. This allows a pool to
-contain a mix of SED and non-SED devices.
-
-A password-protected SED device protects the data stored on the device
-when the device is physically removed from the %brand% system. This allows
-secure disposal of the device without having to first wipe its contents.
-If the device is instead removed to be repurposed on another system, it
-can only be unlocked if the password is known.
-
-.. warning:: It is important to remember the password! Without it, the
-   device is unlockable and its data remains unavailable. While it is
-   possible to specify the PSID number on the label of the device with
-   the :command:`sedutil-cli` command, doing so will erase the contents
-   of the device rather than unlock it. Always record SED passwords
-   whenever they are configured or modified and store them in a safe
-   place!
+`camcontrol <https://www.freebsd.org/cgi/man.cgi?query=camcontrol>`__
+for legacy devices and
+`sedutil-cli <https://www.mankier.com/8/sedutil-cli>`__
+for TCG devices. When managing a SED from the command line, it is
+important to use :command:`sedutil-cli` rather than camcontrol to access
+the full capabilities of the device. %brand% provides the
+:command:`sedhelper` wrapper script to ease SED administration from the
+command line.
 
-When SED devices are detected during system boot, the middleware checks
-for global and device-specific passwords. Devices with their own password
-are unlocked with their password and any remaining devices, without a
-device-specific password, are unlocked using the global password.
+By default, SED are not locked until the administrator takes ownership
+of them. This is done by explicitly configuring a global or per-device
+password in the %brand% |web-ui| and adding the password to the SEDs.
 
-To configure a global password, go to :menuselection:`System -->
-Advanced --> SED Password` and enter the
-password. Recording the password and storing it in a safe place is
-recommended.
+A password-protected SED protects the data stored on the device
+when the device is physically removed from the %brand% system. This
+allows secure disposal of the device without having to first wipe its
+contents. If the device is instead removed to be repurposed on another
```

Comment:
The "instead removed" makes the second sentence confusing.  Well, that, and it's hard to see how "if the password is known" relates to the first sentence.  Please clarify.  Also, the second sentence could be restructured without the if/then pause: "SEDs that will be repurposed on another system can only be unlocked if the password is known."